modules/cluster-management/src/main/java/org/apache/ignite/internal/cluster/management/ClusterManagementGroupManager.java

```diff
@@ -828,8 +792,14 @@ public CompletableFuture<Void> onJoinReady() {
         }
     }
 
-    public CompletableFuture<UpdateDistributedConfigurationAction> clusterConfigurationToUpdate() {
-        return updateDistributedConfigurationActionFuture;
+    /**
+     * Returns a future resolving to the initial cluster configuration. The resulting configuration may be {@code null} if not provided by
+     * the user.
+     *
+     * @return a CompletableFuture that, upon completion, provides the initial cluster configuration, which may be {@code null}.
```

Comment:
There is not enough example in what form the configuration will be returned in string representation.